speechbrain/lobes/models/huggingface_transformers/textencoder.py

```diff
@@ -0,0 +1,125 @@
+"""This lobe enables the integration of generic huggingface pretrained text
+encoders (e.g. BERT).
+
+Transformer from HuggingFace needs to be installed:
+https://huggingface.co/transformers/installation.html
+
+Authors
+ * Sylvain de Langen 2024
+"""
+
+import torch
+import logging
+import os
+
+from speechbrain.lobes.models.huggingface_transformers.huggingface import (
+    HFTransformersInterface,
+)
+
+from typing import Optional
+
+logger = logging.getLogger(__name__)
+
+os.environ["TOKENIZERS_PARALLELISM"] = "false"
```

Comment:
What is that?

Reply:
Good point, this is an artifact from using another file in the module as a reference... It seems to be present for several others, and I don't know what the actual rationale for it was.